Makefile

```diff
@@ -68,7 +68,6 @@ Available targets:
 	               and MacOSX using homebrew)
 	native-deploy: create and start the native deployment
 	deb:           create .deb package for installation on Debian or Ubuntu
-	rpm:           create .rpm package for installation on Fedora-like distros
```

Comment:
There should still be a`make` target for the deployment (even if it just an `rpm` call). I don't undesrtand the rationale for removing it.

Reply:
ACK